packages/core/src/odata/common/request-builder/batch/batch-request-options.ts

```diff
@@ -0,0 +1,13 @@
+import { Destination } from '../../../../scp-cf';
+
+export interface BatchRequestOptions {
+  subrequestPath: 'absolute' | 'servicePathRelative' | 'entityRelative';
+}
+
+export type BatchRequestSerializationOptions = BatchRequestOptions & {
+  destination?: Destination;
```

Comment:
General: I know this is hard to achieve, but is there a way to statically enforce that the destination is given if the obtion `absolute` is set? This would be beneficial if type scrtip users are informed: hey you are doing something which gives an exception.

Reply:
I added this. The code itself became a bit uglier here, but I think it's worth it.